src/Nethermind/Nethermind.Consensus.Clique/CliqueBlockProducer.cs

```diff
@@ -424,7 +424,7 @@ bool IBlockProducer.IsProducingBlocks(ulong? maxProducingInterval)
         _stateProvider.StateRoot = parentHeader.StateRoot!;
 
         IEnumerable<Transaction> selectedTxs = _txSource.GetTransactions(parentBlock.Header, header.GasLimit);
-        Block block = new BlockToProduce(header, selectedTxs, Array.Empty<BlockHeader>());
+        Block block = new BlockToProduce(header, selectedTxs, Array.Empty<BlockHeader>(), Enumerable.Empty<Withdrawal>());
```

Comment:
Enumerable.Empty<Withdrawal>() it should be null probably

Reply:
Well, since Shanghai, null is invalid. Decide based on timestamp?

Reply:
yes, exactly.